src/apple-auth.js

```diff
@@ -139,4 +138,4 @@ class AppleAuth {
 
 }
 
-module.exports = AppleAuth;
+module.exports = { AppleAuth };
```

Comment:
This should be module.exports = AppleAuth

Reply:
But this break ts decorator